documentation/troubleshooting/ipi-install-upstream-troubleshooting.adoc

```diff
@@ -40,12 +51,29 @@ link:Troubleshooting.pdf[Download] the PDF version of this document or visit htt
 ====
 
 
-While attempting to deploy Installer Provisioned Infrastructure (IPI) of
-OpenShift on Bare Metal (BM), you may run into a situation where you need
-to troubleshoot your environment. This document provides troubleshooting
-guidance and tips in solving common issues that may arise.
```

Comment:
Can we not keep this sentence? They remove it because of how their TOC is used downstream. Since our doc is a standalone I think it makes sense to have some intro

Reply:
readded